products/views.py

```diff
@@ -1,3 +1,46 @@
-from django.shortcuts import render
+import re
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Q
+
+from .models          import Product
+
+class ProductOptionView(View):
+    def get(self, request):
+        option = request.GET.get('option', None)
+        offset = request.GET.get('offset', 1)
+        limit  = request.GET.get('limit', 12)
+        order  = request.GET.get('order', None)
```

Comment:
order를 받을 때 front에서 아래와 같이 받고 default를 "id"로 설정해 놓으면 line 16~ 20까지 코드가 필요 없어 보입니다.

```suggestion

        ?ordering=?
        ?ordering=-id
        ?ordering=id
        
        order  = request.GET.get('order', "id")
```

Reply:
이것도 역시 정해진 값으로 보내지 않을 경우를 생각해서 처리했습니다.